Product.md

```diff
@@ -2,32 +2,117 @@
 
 ## Product Manager
```

Comment:
It is unclear to me from this job description if knowing what Octopus is and how it works is a requirement, a goal, or a non-requirement. I think this would be good to call out as a requirement (or non-requirement as the case may be).

Reply:
Added a bullet under the craft as discussed.